clamav/clamav.lua

```
@@ -1,318 +1,297 @@
-		local check, err = has_variable("USE_CLAMAV", "yes")
-		if check == nil then
-			return self:ret(true, "error while checking variable USE_CLAMAV (" .. err .. ")")
-		end
-		if not check then
-			return self:ret(true, "Clamav plugin not enabled")
-		end
+  -- Parse multipart data to extract file parts
+  local parts = self:parse_multipart(body, boundary)
+
+  -- Scan each file part with ClamAV
+  for i, part in ipairs(parts) do
```

Luacheck (1.2.0)

[warning] unused loop variable 'i'

(W213)